functions/src/definitions/common/machineLearningServer/operations.ts

```diff
@@ -40,6 +54,14 @@ async function getL1Category(text: string): Promise<string> {
   return response.data.prediction
 }
 
+async function performOCR(url: string): Promise<OCRResponse> {
+  const data = {
+    url: url,
+  }
+  const response = await callAPI<OCRResponse>("ocr", data)
+  return response.data
```

Comment:
Will be good to map data to camelCase here since the returned value is using snake_case for consistency in other parts of the app

Reply:
Done